reference-architecture/aws-ansible/playbooks/openshift-setup.yaml

```diff
@@ -25,12 +25,11 @@
     openshift_hosted_router_replicas: 3
     openshift_hosted_registry_replicas: 3
     openshift_master_cluster_method: native
-    openshift_node_local_quota_per_fsgroup: 512Mi
```

Comment:
As it is in the installer and the value is the default one, I think it safe to remove it and add support for extra-vars when calling the playbooks/openshift-install.yaml playbook. What do you think?